src/app/Providers/components/CloudAnalyticsInfoAlert.tsx

```diff
@@ -9,7 +9,11 @@ const CloudAnalyticsInfoAlert: React.FunctionComponent = () => {
     LocalStorageKey.isProvidersPageMAAlertHidden
   );
 
-  const link = <a href={CLOUD_MA_LINK.href}>{CLOUD_MA_LINK.label}</a>;
```

Comment:
Looks like this change carried over from your other PR by accident, need to rebase.